### DISCUSSION TITLE
What about support for working with LiteStar?

### DISCUSSION THREAD

**Discussion by pikhovkin** [Lancetnik/Propan#138] [Questions]
What about support for working with [LiteStar](https://litestar.dev/)? [Benchmarks](https://docs.litestar.dev/latest/benchmarks.html#json)

**Comment by Lancetnik** [comment]
If litestar supports lifespan protocol to start and stop broker with Litestar app, Propan can be integrated already
Or are you talking about much close integration like in the FastAPI Router case?
